{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "16e89085_69d69056",
        "filename": "tools/compare_failed_tests/aggregate_reports.py",
        "patchSetId": 5
      },
      "lineNbr": 52,
      "author": {
        "id": 1966697
      },
      "writtenOn": "2023-08-15T07:32:55Z",
      "side": 1,
      "message": "first_report_file",
      "range": {
        "startLine": 52,
        "startChar": 46,
        "endLine": 52,
        "endChar": 61
      },
      "revId": "028f1f2342d198d591226b48a11e93d37565ac10",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "e05d5888_972f073f",
        "filename": "tools/compare_failed_tests/aggregate_reports.py",
        "patchSetId": 5
      },
      "lineNbr": 70,
      "author": {
        "id": 1966697
      },
      "writtenOn": "2023-08-15T07:32:55Z",
      "side": 1,
      "message": "likewise, please make sure that `temp_dir` will be deleted even if exceptions happen.",
      "range": {
        "startLine": 70,
        "startChar": 2,
        "endLine": 70,
        "endChar": 25
      },
      "revId": "028f1f2342d198d591226b48a11e93d37565ac10",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "0d35221b_b3e0dc08",
        "filename": "tools/compare_failed_tests/parse_cts_report.py",
        "patchSetId": 5
      },
      "lineNbr": 51,
      "author": {
        "id": 1954181
      },
      "writtenOn": "2023-08-15T04:54:12Z",
      "side": 1,
      "message": "```suggestion\n    return self.info[\u0027build_fingerprint\u0027] \u003d\u003d info[\u0027build_fingerprint\u0027]\n```",
      "range": {
        "startLine": 48,
        "startChar": 0,
        "endLine": 51,
        "endChar": 18
      },
      "revId": "028f1f2342d198d591226b48a11e93d37565ac10",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "0c8c7757_c9fb9469",
        "filename": "tools/compare_failed_tests/parse_cts_report.py",
        "patchSetId": 5
      },
      "lineNbr": 281,
      "author": {
        "id": 1966697
      },
      "writtenOn": "2023-08-15T07:32:55Z",
      "side": 1,
      "message": "Use try-finally to guarantee that `rmtree` is always called even if exceptions happen. (or use `tempfile.TemporaryDirectory`)\n```\ntry:\n  ...\nfinally:\n  shutil.rmtree(...)\n```",
      "range": {
        "startLine": 267,
        "startChar": 0,
        "endLine": 281,
        "endChar": 25
      },
      "revId": "028f1f2342d198d591226b48a11e93d37565ac10",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}